--- a/src/Makefile.am.orig	2008-01-09 02:50:10.000000000 +0200
+++ b/src/Makefile.am	2008-01-09 02:59:40.000000000 +0200
@@ -84,6 +84,12 @@
          via_xvmc.h 
 endif
 
+pcidatadir = $(datadir)/xserver-xorg/pci
+pcidata_DATA = openchrome.ids
+
+openchrome.ids : via_id.h
+	awk '/^#define.*PCI_CHIP/ {print $$3}' ${srcdir}/via_id.h | sed -e 's/0x/1106/' > openchrome.ids
+
 via_driver.lo: svnversion.h
 svnversion.h:
 	@if [ -f svnrelease.h ]; then \
