Author: Dylan Aïssi
Description: Fix spelling error.
Last-Update: 2016-12-06
Forwarded: https://bugs.freedesktop.org/show_bug.cgi?id=99074

--- a/src/via_xv.c
+++ b/src/via_xv.c
@@ -357,7 +357,7 @@
                 memEfficiency = (float)SINGLE_3205_133;
                 break;
             default:
-                ErrorF("Unknow DRAM Type!\n");
+                ErrorF("Unknown DRAM Type!\n");
                 mClock = 166;
                 memEfficiency = (float)SINGLE_3205_133;
                 break;
