Author: Chris Lamb
Description: Make the build reproducible.
Origin: https://bugs.debian.org/831405
Forwarded: https://bugs.freedesktop.org/show_bug.cgi?id=98896

--- a/src/Makefile.am	2016-07-15 17:56:15.066700891 +0200
--- b/src/Makefile.am	2016-07-15 18:22:08.362042386 +0200
@@ -113,6 +113,9 @@
         if [ -d .svn ]; then \
 	        echo '#define BUILDCOMMENT "(development build, at revision '\
 	                "`svnversion -nc .. | sed -e s/^[^:]*://`"')"' > $@.tmp; \
+        elif [ "$$SOURCE_DATE_EPOCH" ]; then \
+	        printf '#define BUILDCOMMENT "(compiled with SOURCE_DATE_EPOCH: %s)"' $$SOURCE_DATE_EPOCH \
+        	> $@.tmp; \
         else \
 	        date +'#define BUILDCOMMENT "(development build, compiled on %c)"' \
         	> $@.tmp; \
